### PR TITLE
Chore/remove webapp admin again

### PR DESCRIPTION
**Pull request by katywatkins**
cherry-picked the commits from https://github.com/forever-inc/forever-style-guide/pull/163. Looks like it must have been merged into the epic branch *after* the epic was merged into dev.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    forever_style_guide (3.2.3)
```
Comment (alexisBimatovs) [inline Gemfile.lock]: this file should not be commited

File: app/views/forever_style_guide/sections/components/navigation/_nav_account_dropdown_menu.erb [hunk 2]
```diff
@@ -19,13 +19,7 @@
...
           <li class="dropdown-menu-account-item">
             <%= link_to store_admin_url, class: 'dropdown-menu-nav-link dropdown-menu-nav-link-sm dropdown-menu-nav-link-no_margin', title: 'Administer The Forever Store' do %>
-              <i class='fa fa-shopping-basket fa-fw'></i>Store Admin
-            <% end %>
-          </li>
-
-          <li class="dropdown-menu-account-item">
-            <%= link_to web_app_admin_url, class: 'dropdown-menu-nav-link dropdown-menu-nav-link-sm dropdown-menu-nav-link-no_margin', title: 'Administer The Forever Web App' do %>
-              <i class='fa fa-magic fa-fw'></i>Web App Admin
+              <i class='fa fa-shopping-basket fa-fw'></i>Admin
```
Comment (alexisBimatovs) [inline app/views/forever_style_guide/sections/components/navigation/_nav_account_dropdown_menu.erb]: @katywatkins just curious what is the reasoning for the shopping basket being icon here?

**Comment by alexisBimatovs** [comment]
@katywatkins back to you, one small question and one small comment :)

File: Gemfile.lock (hunk 1 above)
Comment (katywatkins) [inline Gemfile.lock]: gemfile.lock is one of the two files we update to change the version. (for reference: https://github.com/forever-inc/forever-style-guide/pull/168/commits/54189dbaa38e92cf47bb90437afc2a6e35e373a0)

**Comment by alexisBimatovs** [comment]
LGTM 👍

**Pull request merged**